app/controllers/bus_stops_controller.rb

```diff
@@ -84,7 +84,10 @@ def update
       redirect_to bus_stops_path
     else
       flash[:errors] = @stop.errors.full_messages
-      render 'edit'
+      @stop.completed = false
+      @stop.completed_by = nil
+      @stop.save
```

Comment:
I'm open to suggestions.

Reply:
I would suggest:

```ruby
@stop.update completed: false
```
Then move this line to the end of the controller method, or even as an `after_action`:

https://github.com/umts/stop-project/blob/d6ebc58a8cde7b840aa7f52f044e2d62a0a6d206/app/controllers/bus_stops_controller.rb#L81

Then the model callback will handle setting the `completed_by`:

https://github.com/umts/stop-project/blob/b83e5e2f2a8cdb39b5eeed28c0094e76c08090a7/app/models/bus_stop.rb#L242-L246